src/test/java/seedu/address/testutil/resident/TypicalResidents.java

```diff
@@ -72,6 +72,14 @@
             .withRoom(VALID_ROOM_BOB)
             .build();
 
+    // First Typical Resident
+    public static final Resident AARON = new ResidentBuilder().withName("Aaron Ang")
+            .withEmail("aaron@example.com")
+            .withPhone("91258294")
+            .withYear("1")
+            .withRoom("01-000")
```

Comment:
Will creating a Resident with a room cause trouble with allocation and deallocation?

Reply:
This resident is used solely for allocation command tests.